components/rules/participantsMenu.tsx

```diff
@@ -1,131 +1,133 @@
-import { PipelineInfo } from "https://deno.land/x/snitch_protos@v0.0.56/protos/info.ts";
-import { Edit, Info, Pause, Silence } from "../icons/crud.tsx";
+import {PipelineInfo} from "https://deno.land/x/snitch_protos@v0.0.56/protos/info.ts";
+import {Edit, Info, Pause, Silence} from "../icons/crud.tsx";
 import IconDots from "tabler-icons/tsx/dots.tsx";
-import { useState } from "https://esm.sh/stable/preact@10.15.1/denonext/hooks.js";
-import { pausePipeline } from "../../lib/fetch.ts";
+import {useState} from "https://esm.sh/stable/preact@10.15.1/denonext/hooks.js";
 
-export const NodeMenu = ({ data }: { data: PipelineInfo }) => {
-  const [isOpen, setIsOpen] = useState();
+export const NodeMenu = ({data}: { data: PipelineInfo }) => {
+    const [isOpen, setIsOpen] = useState();
 
-  const handleClick = () => {
-    setIsOpen(!isOpen);
-  };
-  //todo: pass down envars to browser/move call to route handler
-  //   const pause = (id: string | undefined) => {
-  //     pausePipeline(id);
-  //   };
+    const handleClick = () => {
```

Comment:
Not sure the handle click wrapper adds any value...maybe just call `() => setIsOpen()` in the click handler.